src/neuroglancer_annotation_server/config.py

```diff
@@ -8,10 +8,14 @@ class BaseConfig(object):
     # Statement for enabling the development environment
     DEBUG = True
 
-    NEUROGLANCER_URL = "https://neuroglancer-demo.appspot.com"
+    NEUROGLANCER_URL = "https://fcc-fix2-dot-neuromancer-seung-import.appspot.com"
```

Comment:
This should probably be the seung lab master maybe nkem-multicut.

Reply:
so it can't be those because their neuroglancer python interaction is broken. fcc-fix2 is nkem-multicut with 2 more commits to fix its neuroglancer python interaction.